tests/test_helpers.py

```diff
@@ -51,6 +55,35 @@ def test_generate_coverage_flags_set(
     assert result == expected_flags
 
 
+@pytest.mark.parametrize(
+    "value",
+    [
+        pytest.param("vllm", id="value:vllm"),
+        pytest.param("", id="empty"),
+        pytest.param("1", id="1"),
+        pytest.param("0", id="0"),
```

Comment:
Just a question that what the values of 0, -1, 2 will be used for?

Reply:
Only the value `1` is valid/accepted to enable the feature, all other values used in this test are to ensure it does not do anything at all/leave it disabled.